src/player.js

```diff
@@ -1,26 +1,23 @@
-export class Player{
-  constructor(newID){//finish reading on WeakMaps for private props
-    this.playerID = newID;
-    this.postition = null;
+export default class Player{
+  constructor(name, regID, socketID){//finish reading on WeakMaps for private props
+    this.person = name;
+    this.registeredID = regID;
+    this.socketID = socketID;
+    this.position = null;
     this.money = null;
-    this.propertiesOwned = new Array(28);
-    this.cardsOwned = null; //change to list or something;
+    this.propertiesOwned = new Array(0);
+    this.jailFreeCards = null; //change to list or something;
     this.isJailed = false;
+    this.doubles = null;//
   }
 
-  static transferMoney(FromPlayer, ToPlayer, amount){
-    if(amount > FromPlayer){
-      //throw error, inssuficient funds;
+  movePlayer(diceArray, boardLength){//DONE
+    this.position += diceArray.reduce(( acc, cur ) => acc + cur, 0);
```

Comment:
You actually do this pattern a few times.  If this continues you might consider building a new `Dice` object that has access to the individual rolls in addition to a nice `.sum()` function.